Kudu.Services.Web/App_Start/NinjectServices.cs

```diff
@@ -731,5 +730,19 @@ private static IEnvironment GetEnvironment(IDeploymentSettingsManager settings =
             string binPath = HttpRuntime.BinDirectory;
             return new Core.Environment(root, EnvironmentHelper.NormalizeBinPath(binPath), repositoryPath);
         }
+
+        private static void EnsureDotNetCoreEnvironmentVariable()
+        {
+            SetEnvironmentVariableIfNotYetSet("DOTNET_SKIP_FIRST_TIME_EXPERIENCE", "true");
```

Comment:
Add comment:

    // Skip this as it causes huge files to be downloaded to the temp folder